packages/core/admin/_internal/node/develop.ts

```diff
@@ -47,6 +80,12 @@ const develop = async ({
       return;
     }
 
+    // Build without diagnostics in case schemas have changed
+    if (distDir) {
```

Comment:
Shouldn't we explicitly check that the `tsconfig` exists (on top of checking the `distDir`) to make it transparent that we're checking whether or not we're in a TS project?
(I know that the `distDir` is defined only if the `tsconfig` exists, but that feels weird for some reason. What do you think?)